docs/source/examples.rst

```diff
@@ -180,3 +180,22 @@ specified for certain steps in the processing. For example:
     result_stage_2 = client.compute(stage_2,
                                     resources={tuple(stage_1): {'GPU': 1},
                                                tuple(stage_2): {'ssdGB': 100}})
+Remote SLURM Deployment
+-----------------
+
+All the above SLURMCluster examples are also possible over the SLURM REST API with the use of
+RemoteSlurmCluster. See https://slurm.schedmd.com/rest_api.html for more information.
```

Comment:
This is only likely to work under pretty restricted circumstances (like you mentioned in https://github.com/dask/dask-jobqueue/issues/502#issuecomment-874697650) so it is probably good to document that. Something like
```
Remote SLURM deployment is only going to work when the scheduler has access to the SLURM REST API _and_ the workers that are launched on the (remote) cluster are able to communicate with the scheduler over HTTP (e.g., the scheduler is inside a container that is running on the cluster where the jobs will be launched).
```

Reply:
Added